### ISSUE TITLE
Running tests hanging forever

### ISSUE THREAD

**Issue by lowki** [serverless-stack/sst#2709]
Testing with latest version, from `create-sst` command, tests are running forever

Steps to reproduce.
```shell
npx create-sst@latest sst-test
cd sst-test
pnpm install
cd packages/core
pnpm test
```

![image](https://user-images.githubusercontent.com/1659775/227907931-b0c72000-8760-4e8e-a731-c06b650b96b3.png)

Running vitest without sst bind exit as expected.
![image](https://user-images.githubusercontent.com/1659775/227908547-e718d1e5-d8f7-40dc-b84c-08f1fcfd6e6d.png)

**Comment by bjornlll** [comment]
Anything executed with `sst bind` seems hang and never terminate.

**Comment by bjornlll** [comment]
Downgrading to `sst@2.2.8` solved it for me.

**Comment by lowki** [comment]
> Anything executed with `sst bind` seems hang and never terminate.

Is this an expected behaviour?

**Comment by fwang** [comment]
@lowki @bjornlll thanks for bringing this up!

Fixed in v2.3.2.
